core/rawdb/accessors_chain.go

```diff
@@ -284,6 +288,16 @@ func ReadCurrentHeader(db kv.Getter) *types.Header {
 	return ReadHeader(db, headHash, *headNumber)
 }
 
+// ReadCurrentHeader2 reads the current canonical head header.
+func ReadCurrentHeader2(db kv.Getter) *types.Header {
```

Comment:
rename to ReadCurrentBlockHeader?

Reply:
I'd like to understand the difference between the 2 tables before renaming. I've asked on discord.